addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -267,6 +346,248 @@ odoo.define('website_slides.quiz', function (require) {
                 this._alertShow();
             }
         },
+
+        _onQuizCreation: function () {
+            var $elem = this.$('.o_wslides_js_quiz_add');
+            new QuestionWidget(this, null, null, this.slide.id, this.quiz.questions.length + 1).replace($elem);
+        },
+
+        _onEditQuestionClick: function (ev) {
+            var $elem = $(ev.currentTarget).closest('.o_wslides_js_lesson_quiz_question');
+            new QuestionWidget(this, $elem, this._getQuestionDetails($elem), this.slide.id).replace($elem);
+        },
+
+        _onDeleteQuestionClick: function (ev) {
+            var question = $(ev.currentTarget).closest('.o_wslides_js_lesson_quiz_question');
+            new ConfirmationDialog(this, null, question).open();
+        },
+
+        _displayCreatedQuestion: function (response) {
+            var $lastQuestion = this.$('.o_wslides_js_lesson_quiz_question:last');
+            if ($lastQuestion[0]) {
+                $lastQuestion.after(QWeb.render('slide.quiz.question.view', {question: response.data}));
+            } else {
+                this.$el.prepend(QWeb.render('slide.quiz.question.view', {question: response.data}));
+            }
+            this.quiz.questions.push({
+                id: response.data.id,
+                title: response.data.question,
+                answers: response.data.answer_ids.map(obj => { return {id: obj.id, text: obj.text_value} })
+            });
+            var $elem = this.$('.o_wsildes_quiz_question_input');
+            if (response.data.createNext) {
+                new QuestionWidget(this, null, null, this.slide.id, this.quiz.questions.length + 1).replace($elem);
+            } else {
+                $elem.replaceWith(QWeb.render('slide.quiz.new.question.button'));
+            }
+        },
+
+        _displayUpdatedQuestion: function (response) {
+            response.data.$elem.html(QWeb.render('slide.quiz.question.view', { question:response.data }));
+        },
+
+        _resetDisplay: function (reset) {
```

Comment:
Not sure why but this looks a little overkill.
I will have to take a second look after testing functionally.